sdk/core/core-http/src/util/exponentialBackOffRetry.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
minor suggestion: 'exponential' is sometimes referred to as a retry "strategy" so this util could also be `exponentialRetryStrategy.ts`

Reply:
(Technically we're supposed to support linear retry per the guidelines but we don't)

Reply:
Fixed